code/modules/gear_presets/upp.dm

```diff
@@ -3637,6 +3637,106 @@
 		list("Large General Pouch", 15, /obj/item/storage/pouch/general/large, null, VENDOR_ITEM_REGULAR),
 	)
 
+/datum/job/antag/upp/pilot
+	title = JOB_UPP_PILOT
+	gear_preset = /datum/equipment_preset/upp/pilot
+
+/datum/equipment_preset/upp/pilot
+	name = "UPP Pilot (Cryo)"
+	assignment = JOB_UPP_PILOT
+	role_comm_title = "Pilot"
+	rank = JOB_UPP_PILOT
+	paygrades = list(PAY_SHORT_UO1 = JOB_PLAYTIME_TIER_0)
+	skills = /datum/skills/pilot
+	access = list(ACCESS_UPP_GENERAL, ACCESS_UPP_ENGINEERING, ACCESS_UPP_FLIGHT)
+
+/datum/equipment_preset/upp/pilot/dressed
+	name = "UPP Pilot"
+	flags = EQUIPMENT_PRESET_EXTRA
+
+/datum/equipment_preset/upp/pilot/load_gear(mob/living/carbon/human/new_human)
```

Comment:
Wouldn't that be on /dressed ?

Reply:
The UPP Pilot role follows the same setup to other shipside UPP roles wherein their "cryo" role has identical equipment presets to the dressed role, the main difference is that the cryo role is required for the role to appear as a selectable role for the Faction Clash game mode. 

More differentiation may be possible in the future but due to the sprite freeze there is an inability to add new gear.

Reply:
Ah, well, okay...